src/Event/Event.php

```diff
@@ -157,7 +157,7 @@ public static function off($event)
     public static function bound($event)
     {
         $onces = isset(static::$events['__bow.once.event']) ? static::$events['__bow.once.event'] : [];
-        $translations = isset(static::$events['__bow.transmission.event']) ? static::$events['__bow.transmission.event'] : [];
+        $translations = isset(static::$events['__bow.transmission.event']) ? static::$events['__bow.transmission.event'] : []; // phpcs:ignore
```

Comment:
The following characters are greater than the `120` lengths in the PSR-2 coding style.

To ignore this line, adding the `// phpcs:disable` comment.